deploy/src/main/resources/production-swarm/README.md

```diff
@@ -19,6 +19,8 @@ The `environment.sh` file supports configurable property settings necessary for
 	###
 	# Elasticsearch
 	###
+	## The protocol used to connect with elasticsearch server.
```

Comment:
The protocol used to connect **to the Elasticsearch server**.

Reply:
Updated description.